src/zshrc/_pcp

```diff
@@ -183,6 +183,9 @@ _pcp () {
       "(-m --include-labels -m --include-labels $exargs)"{-m,--include-labels}'[include labels]' \
       "(-n --invert-filter $exargs)"{-n,--invert-filter}'[perform ranking before live filtering]' \
       "(-N --predicate $exargs)"{-N+,--predicate=}'[specify predicate metric]:metric:->metrics' \
+      "(-u --url $exargs)"{-u+,--url=}'[URL of endpoint to receive HTTP POST]' \
+      "(-U --http-user $exargs)"{-U+,--http-user=}'[username for HTTP basic authentication]' \
+      "(-p --http-pass $exargs)"{-p+,--http-pass=}'[password for HTTP basic authentication]' \
```

Comment:
A nit-pick comment here, earlier these HTTP POST / URL related params were added after the other tool-specific params but here they are added before them, ideally would consistently add always after the earlier params but no biggie.

Reply:
Fixed, thanks.